IntelliTect.TestTools.Selenate/GoogleSearch/ExampleTests.cs

```diff
@@ -44,17 +44,38 @@ public void ReturnToHomepage()
                 "Search results displayed when they were not expected");
         }
 
-        [TestCleanup]
-        public void Teardown()
+        [TestMethod]
+        public void TakeScreenshotSavesFile()
         {
-            // Two ways to take screenshots.
+            string path = Path.Combine("C:", "temp", "screenshots");
```

Comment:
Rather than using hard coded paths. It also doesn't guarantee you will have write permissions on the directory. I would suggest leveraging something more like:
```suggestion
            string path = Path.Combine(Path.GetTempPath(), Path.GetRandomFileName());
```

Though I realize the base method relies on this path, so it might not be a viable option.

Reply:
@Keboo Okay, that's pretty neat. :) I figure C:\temp would be a decent bet for being able to write to. Although since TakeScreenshot() already attempts to write there, I suppose this one I could specifically change to write to the current directory.